spec/model_renamer/rename_spec.rb

```diff
@@ -1,3 +1,16 @@
 describe Rename do
-  pending "write it"
+  around do |example|
+    MemFs.activate { example.run }
+  end
```

Comment:
Thoughts about moving this to `spec_helper`?

```
RSpec.configure do |config|
  config.before do
    MemFs.activate!
  end

  config.after do
    MemFs.deactivate!
  end
end
```

Reply:
Yes.. i think that will eventually make sense. Thought i would leave it here for now though.